docs-ref-conceptual/use-cli-effectively.md

```diff
@@ -148,7 +148,7 @@ Many commands offer a `--no-wait` parameter, which allows other commands to run.
 az group delete --name MyResourceGroup --no-wait
 ```
 
-When you remove a resource group, you also remove all the resources that belong to it. Reclaiming these resources can take a long time. The `--no-wait` parameter When this command runs, the console can accept commands immediately, even though it's still working on the command.
+When you remove a resource group, you also remove all the resources that belong to it. Reclaiming these resources can take a long time. When this command is run with the `--no-wait` parameter, the console can accept new commands immediately, even though it's still working on this command.
```

Comment:
```suggestion
When deleting a resource group, all the resources that belong to it are also removed. Removing these resources can take a long time. Running the command with the `--no-wait` parameter, allows the console to accept new commands without interrupting the removal.
```

This suggestion changes the voice to match the rest of the article. Do you think would still satisfy your intent?

Reply:
Sure! Thanks!